### PR TITLE
Add Story type to example

### PR DESCRIPTION
**Pull request by cabello**
Issue: the example does not tell where to import `Story` from.

## What I did

Added the import.

## How to test

- Is this testable with Jest or Chromatic screenshots? No
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: docs/snippets/react/button-story-with-args.ts.mdx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ```ts
 // Button.stories.ts
 
+import { Story } from '@storybook/react';
```
Comment (shilman) [inline docs/snippets/react/button-story-with-args.ts.mdx:4]:
```suggestion
import { Story } from '@storybook/react/types-6-0';
```

**Review by shilman — CHANGES_REQUESTED** [review]
Thanks for the fix! We're using `types-6-0` for now. We're not sure whether we'll need to make breaking changes in 6.1 so `@storybook/react` types are experimental.

**Comment by cabello** [comment]
Thank you for the suggestion, addressed it.

**Review by shilman — APPROVED** [review]
Thanks for the fix! 🙏

**Pull request merged**